app/javascript/dashboard/components/widgets/conversation/VariableList.vue

```diff
@@ -16,6 +16,13 @@ export default {
     ...mapGetters({
       customAttributes: 'attributes/getAttributes',
     }),
+    sanitizedSearchKey() {
```

Comment:
Please move to a helper and add tests cases for the same?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/11864/commits/1aafbac117e393a9d8525e813030dd0c6eab9bb7